packages/grafana-ui/src/components/Link/TextLink.tsx

```diff
@@ -29,7 +29,7 @@ interface TextLinkProps extends Omit<AnchorHTMLAttributes<HTMLAnchorElement>, 't
 }
 
 const svgSizes: {
-  [key in keyof ThemeTypographyVariantTypes]: IconSize;
+  [key in keyof Omit<ThemeTypographyVariantTypes, 'code'>]: IconSize;
```

Comment:
Could create a type at the top for this. Like `type TextLinkVariants = keyof Omit<ThemeTypographyVariantTypes, 'code'>;` and then we could use it both here and in line 23?